docs/table_design/Data_distribution.md

```diff
@@ -112,51 +214,208 @@ CREATE TABLE site_access(
     pv BIGINT SUM DEFAULT '0'
 )
 AGGREGATE KEY(event_day, site_id, city_code, user_name)
--- Set the partitioning method as Range partitioning.
-PARTITION BY RANGE(event_day) (
+PARTITION BY RANGE(event_day)(
     PARTITION p1 VALUES LESS THAN ("2020-01-31"),
     PARTITION p2 VALUES LESS THAN ("2020-02-29"),
     PARTITION p3 VALUES LESS THAN ("2020-03-31")
 )
--- Set the bucketing method as Hash bucketing and specify the bucketing key.
-DISTRIBUTED BY HASH(event_day, site_id);
+DISTRIBUTED BY HASH(site_id);
+```
+
+**Create multiple partitions at a time**
+
+Multiple partitions can be created at a time at and after table creation. You can specify the start and end time for all the partitions created at a time in `START()` and `END()` and the partition increment value in `EVERY()`. However, note that the range of partitions is right hand half open, which includes the start time but does not include the end time. The naming rule for partitions is the same as that of dynamic partitioning.
+
+- **Partition a table on a date-type column (DATE and DATETIME) at the table creation**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/33f53583-bcba-4b11-a02a-75f868fcb877)
remove "the"